.github/workflows/release.yml

```diff
@@ -4,7 +4,7 @@ on:
     inputs:
       formula:
         description: 'Formula to release Homebrew'
-        default: twilio
+        default: twilio@rc
```

Comment:
Does this mean for the production release, we'll need to manually enter `twilio` in input params? If yes, can we update the release doc?

Reply:
Actually, I'll revert this. We can keep the default as Twilio and input the rc parameter when we do a pre release.